magento/pwa-studio
# [PWA-589] Page Reset on Sort

## Description

If a customer changes the sort criteria they would be returned to the first 'page' of the newly sorted items.

## Related Issue





https://jira.corp.magento.com/browse/PWA-589

## Acceptance 



### Verification Stakeholders

@dhaecker 
@jimbo 

### Specification



### Verification Steps


1. Go to a Category/Search results page
2. Sort by Price: High to Low
3. Go to page 2
4. Change the sort to Price: Low to High
5. See that you sent back to page 1

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
